packages/sdk-lsk/source/transaction-type.service.ts

```diff
@@ -70,4 +70,12 @@ export class TransactionTypeService {
 
 		return parseInt(data.type) === 4;
 	}
+
+	public static isUnlockToken(data: TransactionData): boolean {
+		if (isTest(data)) {
+			return data.moduleAssetName === "dpos:unlockToken";
+		}
+
+		return parseInt(data.type) === 5;
```

Comment:
What is this based on? Unlock token didn't exist before their 3.0 release.

Reply:
In other is* methods that number matches the moduleID, so I looked for the moduleID of unlockToken transaction type. However, I am still a bit confused: docs shows that 5, 3 (moduleID, assetID) is for unlockToken https://lisk.com/documentation/lisk-sdk/dpos-module.html#unlock but then in the code seems to be mapped to 5, 2 https://github.com/PayvoHQ/sdk/blob/master/packages/sdk-lsk/source/fee.service.ts#L111-L117

Reply:
In other methods these match the 2.0 transaction types because the `isTest` early return isn't triggered. This method should return false if `isTest` is false because then it isn't a 3.0 transaction.

Reply:
I see. I wasn't aware they had 2.0 for testnet and 3.0 for mainnet. How is this differentiation done in the tests? They seem to cover only 2.0.

Reply:
https://github.com/PayvoHQ/sdk/pull/211/files this might make it more clear by inverting the logic.